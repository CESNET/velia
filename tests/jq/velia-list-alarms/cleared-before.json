EDFA-LINE-SDN-ID210512.json