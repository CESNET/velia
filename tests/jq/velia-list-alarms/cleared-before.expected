"\tResource\tSeverity\tDetail\tTimestamp\tStatus"
"‚è∏\tcla-sdn-inline.service\tcleared\tsystemd unit state: (active, running)\t2024-10-16T13:44:33.123368627+00:00\tcleared"
" \t\tcritical\tsystemd unit state: (activating, auto-restart-queued)\t2024-10-16T13:44:32.123721368+00:00\t"
" \t\tcritical\tsystemd unit state: (inactive, dead)\t2024-10-16T13:44:31.123368627+00:00\t"
